pkg/bootstrap/config.go

```diff
@@ -500,12 +500,6 @@ func getNodeMetaData(envs []string, plat platform.Environment, nodeIPs []string,
 
 	// sds is enabled by default
 	meta.SdsEnabled = true
-	meta.SdsTrustJwt = true
-
-	// Add STS port into node metadata if it is not 0.
```

Comment:
@JimmyCYJ can you confirm this is not needed? I don't see how it can be used but want to verify

Reply:
I don't see this meta.StsPort being accessed directly anywhere. Perhaps this is because the Proxy reads the metadata from config file not from this metadata struct. @bianpengyuan added this to let Proxy read metadata to decide which token credential is needed for gRPC connection.

Reply:
https://github.com/istio/istio/pull/20595 adds this into metadata. I think we need to keep this.

Reply:
thanks, ill add it back

Reply:
I added it back